src/stories/containers/Finances/components/FinacesTable/FinancesTable.tsx

```diff
@@ -44,44 +47,50 @@ const FinancesTable: React.FC<Props> = ({ className, breakdownTable, metrics, pe
       {showFooterAndCorrectNumber.map((table: TableFinances, index) => (
         <TableContainer isLight={isLight} className={className} key={index} hasOthers={table.others || false}>
           <TableBody isLight={isLight}>
-            {table.rows.map((row: ItemRow, index) => (
-              <TableRow key={index} isLight={isLight} isMain={row.isMain}>
-                <Headed isLight={isLight} period={period}>
-                  {row.name}
-                </Headed>
+            {table.rows.map((row: ItemRow, index) => {
+              console.log('row.codePath', row.codePath?.replace('atlas', ''));
+              const href = `${siteRoutes.finances((row.codePath ?? '').replace('atlas/', ''))}?year=${year}`;
```

Comment:
please, note that in the first level the code path is `atlas` so it is not going to be replaced as `atlas !== atlas/` so you should handle this in order to get the right link

Reply:
Sorry I don't got your point, the links start from that level and then going down, so I don't think that affect the navigations, but not sure if I got your point